src/api/base.tsx

```diff
@@ -0,0 +1,14 @@
+import { create } from "apisauce";
+
+const api = create({
+  baseURL: "https://",
+  headers: {
+    // eslint-disable-next-line prettier/prettier
+    Connection: "keep-alive",
+    "Accept-Language": "en-US,en;q=0.8",
+  },
+  // 10 seconds timeout
```

Comment:
Can remove this comment, pretty self explanatory

Reply:
resolved